e2e/files.spec.ts

```diff
@@ -344,72 +344,85 @@ test.describe('Files', () => {
     },
   )
 
-  testAuth(
-    'should create and delete a file',
-    async ({ userPage, browserName }) => {
```

Comment:
I removed `browserName`, since it wasn't used. This resulted in many whitespace changes, so the diff isn't very helpful -- no other changes were made in this file

Reply:
Thanks, this was a catastrophic diff